qcfractal/interface/models/rest_models.py

```diff
@@ -1081,35 +1081,62 @@ class QueueManagerGETBody(BaseModel):
     class Data(BaseModel):
         limit: int = Schema(
             ...,
-            description="Max number of Queue Managers to get from the server"  # TODO: Verify this.
+            description="Max number of Queue Managers to get from the server"
         )
 
-    meta: QueueManagerMeta
-    data: Data
+    meta: QueueManagerMeta = Schema(
+        ...,
+        description="Validation and identification Meta information for the Queue Manager's communication with the "
+                    "Fractal Server"
+    )
+    data: Data = Schema(
+        ...,
+        description="A model of Task request data for the Queue Manager to fetch. Accepts ``limit`` as the maximum "
+                    "number of tasks to pull."
+    )
 
     class Config(RESTConfig):
         pass
 
 
 class QueueManagerGETResponse(BaseModel):
-    meta: ResponseGETMeta
-    data: List[Dict[str, Any]]
+    meta: ResponseGETMeta = Schema(
+        ...,
+        description="Standard metadata returned for fetch queries."
```

Comment:
This might be another spot where `description=ResponseGETMeta.__doc__` would be optimal.